zerver/webhooks/ansibletower/doc.md

```diff
@@ -0,0 +1,19 @@
+Get Ansible Tower notifications in Zulip!
+
+ 1. {!create-stream.md!}
+
+ 1. {!create-bot-construct-url-indented.md!}
+
+ 1. Go to your Ansible Tower or AWX Admin Portal, and click **Notifications** on the left sidebar.
+    Click **Add**.
```

Comment:
Okay, so the first sentence got a bit wordy once we added the " on the left sidebar" stuff.

I'd do the following:
```
Go to your Ansible Tower or AWX Admin Portal. Click **Notifications** on
the left sidebar, and click **Add**.
```